dataactbroker/handlers/fileHandler.py

```diff
@@ -552,6 +552,9 @@ def load_d_file(self, url, upload_name, timestamped_name, job_id, is_local, resp
 
             logger.debug('Marking job id of %s', job_id)
             mark_job_status(job_id, "finished")
+            # only need to close it if we threaded it in the first place
+            if not self.isLocal:
```

Comment:
The reason I didn't put it in a "finally" is because both the try and the except raise/return something, which means the code wouldn't get to the finally right? I can remove the "if" check though